docs/reference/filters.md

```diff
@@ -1192,6 +1192,21 @@ Read client-id and client-secret from environment variables
 secureOauthTokenintrospectionAllKV("issuerURL", "", "", "k1", "v1", "k2", "v2")
 ```
 
+## jwtValidation
+
+The filter parses bearer jwt token from Authorization header and validates the signature using public keys
+discovered via /.well-known/openid-configuratioon endpoint. Takes issuer url as single parameter. 
+The filter stores token palyoad into the state bag where it can be used by oidcClaimsQuery() or forwardTokenPart()
+ 
+
+Examples:
+
+```
+jwtValidation("issuerURL")
```

Comment:
Lets provide some working example - it is not clear from the docs whether `/.well-known/openid-configuration` should be part of this url or not.